lib/src/core/room.dart

```diff
@@ -264,7 +264,7 @@ class Room extends DisposableChangeNotifier with EventsEmittable<RoomEvent> {
         (event) => _onSignalConnectionQualityUpdateEvent(event.updates))
     ..on<SignalStreamStateUpdatedEvent>(
         (event) => _onSignalStreamStateUpdateEvent(event.updates))
-    ..on<SignalSubscribedQualityUpdatedEvent>((event) {
+    ..on<SignalSubscribedQualityUpdatedEvent>((event) async {
       // Dynacast is off or is unsupported
       if (!roomOptions.dynacast || _serverVersion == '0.15.1') {
```

Comment:
with multi-codec simulcast, we need to apply these updates regardless of if dynacast is enabled.

Reply:
https://github.com/livekit/client-sdk-flutter/pull/323/commits/ffea9697afce536c679fcb0171a529bac34769fc